docs/manual-doc/statement-syntax/data-manipulation-statements/pipe-syntax.md

```diff
@@ -0,0 +1,19 @@
+# PIPE-syntax
+
+One major difference between nGQL and SQL is how subqueries are composed.
+
+In SQL, sub-queries are nested (embedded) to form a statement.
+MeanWhile, nGQL uses shell style `PIPE (|)`.
+
+### Examples
+
+```
+GO FROM 201 OVER edge_serve | GO FROM $-.id OVER edge_fans | GO FROM $-.id ...
+
+GO FROM 100 OVER like YIELD like._dst AS dstid, $$.player.name AS Name  | GO FROM $-.dstid OVER like YIELD like._dst, like.likeness, $-.Name
+```
+
+The alias name right after placeholder `$-.` must be either exactly `id` or defined in the previews statement (e.g., `distid` or `Name` as show in the above example).
```

Comment:
The dest `id` would be given by default when `yield` is not declared. If `yield` is declared explicitly, `id` would not be given unless user declares explicitly.

Reply:
After some consideration, I choose not to mention this short-cut-syntax, as too much for the first time to read